packages/language-server/src/tokens/get-token.ts

```diff
@@ -4,6 +4,15 @@ import { color2kToVsCodeColor } from './color2k-to-vscode-color'
 import { isColor } from './is-color'
 import { getTokensInString, hasTokenRef } from './expand-token-fn'
 
+const ESCAPE_HATCHED = /^(\[)(.*)(\])$/
+
+const removeEscapeHatch = (value: string) => {
+  if (ESCAPE_HATCHED.test(value)) {
+    return value.match(ESCAPE_HATCHED)?.[2] as string
```

Comment:
If it matches, the second group should always be available

`[#fff]`
0: `[#fff]`
1: `[`
2: `#fff`
3: `]`